README.md

```diff
@@ -18,15 +18,22 @@ npm run dev
 
 This will start the micro HTTP service on PORT 3000.
 
-## API Example
+## API
 
-```bash
-http://localhost:3000/?id=tt1477834
```

Comment:
I'd add this to the readme if you mention the example response below. It's a good example.